### ISSUE TITLE
" error setting certificate verify locations" for https fetch on XP

### ISSUE THREAD

**Issue by PhilipOakley** [git-for-windows/git#168]
The https fetch now fails with a certificate error, while a git:// fetch succeeds.

Quick test of: 

      $ git --version
      git version 2.4.2.windows.1
      $ uname -a
      MINGW32_NT-5.1 PhilipOakley 2.1.2(0.288/5/3) 2015-05-25 16:03 i686 Msys

(It's XP !)

gave:

      Philip@PhilipOakley MINGW32 /c/msysgit195/git (msvc-PR318-2b)
      $ git fetch https://github.com/dscho/git.git
      fatal: unable to access 'https://github.com/dscho/git.git/': error setting certificate verify locations:
        CAfile: C:/Program Files/Git/mingw32/libexec/ssl/certs/ca-bundle.crt
        CApath: none

    Philip@PhilipOakley MINGW32 /c/msysgit195/git (msvc-PR318-2b)
    $ git fetch git://github.com/dscho/git.git
    From git://github.com/dscho/git
     * branch            HEAD       -> FETCH_HEAD

I previously had Git-2.4.0.2-dev-preview-32-bit.exe installed and it worked OK then (I have a scripted update that was working until minutes before the install/update).

There is no /ssl/ sub-directory to C:/Program Files/Git/mingw32/libexec in my install.

Do others see this difference between protocols (and their security setup)?, and is it only on my XP m/c?

**Comment by dscho** [comment]
Ah, darn. You know what happened? When I reworked the Windows-wide config based on @kblees' Pull Request, I broke XP support (the `getenv()` way to obtain the `%PROGRAMDATA%` value has an XP-only bug). I am on it.

**Comment by dscho** [comment]
Testing https://github.com/dscho/git/compare/git-for-windows:master...xp-fix

**Comment by dscho** [comment]
My test confirmed that it works now.

**Comment by PhilipOakley** [comment]
Ah, darn. You know what happened? When I reworked the Windows-wide config based on @kblees' Pull Request, I broke XP support (the getenv() way to obtain the %PROGRAMDATA% value has an XP-only bug). I am on it.

  Testing dscho@git-for-windows:master...xp-fix

Just did a clean download of the latest SDK, and it's fixed in that (i.e. my BringGitUptodate.sh script) works just fine again. 

I did not get chance to create an insteller and try that version, so there's still a minor possiblility of some other nuance, but I consider it *tested*. 

Many thanks Philip
